pagopa/io-app
# chore(IT Wallet): [SIW-1319] Wallet Instance lifecycle reducer

> [!Warning]
> This PR depends on #5861 

## Short description
This PR adds a lifecycle reducer to IT Wallet. The goal of this reducer is to persist the wallet instance state (installed, operational, valid, deactivated), so it is available in the global store.

## List of changes proposed in this pull request
- Add persisted lifecycle ITW reducer
- Update the wallet instance state from the eID issuance machine
- Refactor persist logic

## How to test
Run the app and check that `features.itWallet.lifecycle` gets updated after the eID issuance.
